### ISSUE TITLE
page populator for checkbox that is belong to iframe

### ISSUE THREAD

**Issue by vveliev** [cheezy/page-object#360]
When populating checkbox that is belong to the iframe 'enabled?' validation is failing.

```
element.enabled?
*** Selenium::WebDriver::Error::StaleElementReferenceError Exception: Element belongs to a different frame than the current one - switch to its containing frame to use it
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.52.0', revision: '4c2593c', time: '2016-02-11 19:06:42'

```

**Comment by ducthanh** [comment]
maybe you need to switch to the enabled frame before verifying

**Comment by vveliev** [comment]
maybe I don't 

The problem is that basic methods have no reference element when they do validation, as the example,
I can check/uncheck it, but I cannot validate my action.

**Comment by cheezy** [comment]
What would you want to have happen here?

**Comment by vveliev** [comment]
in perfect world i would want it to work, in reality I think it should save some reference to iframe so that at the time of validation it will pass.

**Comment by cheezy** [comment]
Is the checkbox declared inside of a frame using the `in_iframe` method?

**Comment by vveliev** [comment]
yes 
```rb
          in_iframe(id: 'preferences-frame') do |frame|
            checkbox(:show_preview, id: 'rcmfd_preview', frame: frame)
            select_list(:mark_preview, id: 'rcmfd_preview_pane_mark_read', frame: frame)
            select_list(:on_requersm, id: 'rcmfd_mdn_requests', frame: frame)
            select_list(:expand, id: 'rcmfd_autoexpand_threads', frame: frame)
            text_field(:rows_per_page, id: 'rcmfd_mail_pagesize', frame: frame)
            checkbox(:all_folders_new_message, id: 'rcmfd_check_all_folders', frame: frame)
            button(:save, id: 'rcmbtn103', frame: frame)
          end
```

**Comment by AlexisKAndersen** [comment]
@vveliev, is this issue still affecting you? If so, can you provide examples I can use to duplicate the issue?

**Comment by vveliev** [comment]
I will need to find this code .... :) , but i think it's been fixed after version 2.0.0
